versions.tf

```diff
@@ -0,0 +1,7 @@
+terraform {
+  required_version = ">= 0.12"
+
+  required_providers {
+    google = ">= 3.0, <4.0.0"
```

Comment:
Isn't `"~> 3.0" == ">=3.0, <4.0.0"`? I think that should cover the version constraint.